test/framework/modules.py

```diff
@@ -1091,6 +1091,7 @@ def test_module_caches(self):
 
         # purposely extending $MODULEPATH with non-existing path, should be handled fine
         nonpath = os.path.join(self.test_prefix, 'nosuchfileordirectory')
+        mkdir(nonpath)
```

Comment:
The comment above no longer makes sense since it is no longer a non-existing path.